style.css

```diff
@@ -0,0 +1,37 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    color: rgb(14, 166, 241);
+}
+
+body{
+    display: flex;
+    justify-content: center;
+    align-self: center;
+
+    min-height: 100vh;background: #222;
+}
+.container{
+    position: relative;
+    max-width: 100%;
+
+    display: grid;
+    grid-template-columns: repeat(auto-fill,minmax(300px,1fr));
+    grid-template-columns: repeat(100px,auto);
```

Comment:
grid-template-columns or grid-template-rows    --->    check once

Reply:
yes, its rows in 2nd line